src/tree/SlotTreeItemBase.ts

```diff
@@ -106,8 +104,14 @@ export abstract class SlotTreeItemBase extends AzureParentTreeItem<ISiteTreeRoot
             case DeploymentTreeItem.contextValue:
                 return this.deploymentsNode;
             default:
-                // DeploymentTreeItem.contextValue is a RegExp, but the passed in contextValue can be a string so check for a match
-                if (typeof expectedContextValue === 'string' && DeploymentTreeItem.contextValue.test(expectedContextValue)) { return this.deploymentsNode; }
+                if (typeof expectedContextValue === 'string') {
+                    // DeploymentTreeItem.contextValue is a RegExp, but the passed in contextValue can be a string so check for a match
+                    if (DeploymentTreeItem.contextValue.test(expectedContextValue)) {
+                        return this.deploymentsNode;
+                    }
+                } else if (expectedContextValue.source.includes(FunctionTreeItem.contextValueBase)) {
```

Comment:
Is this returning the parent? I'm not sure what the source is here.

Reply:
source is the RegExp as a string. Aka it's converting the regexp to a string and seeing if it has `azFuncFunction`